packages/cli/src/commands/BaseCommand.ts

```diff
@@ -44,7 +44,9 @@ export abstract class BaseCommand extends Command {
 	 * How long to wait for graceful shutdown before force killing the process.
 	 * Subclasses can override this value.
 	 */
-	protected gracefulShutdownTimeoutInS: number = 30;
+	protected gracefulShutdownTimeoutInS: number = config.get(
```

Comment:
You can use `getEnv` to infer the type, ref. https://github.com/n8n-io/n8n/pull/2656